Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
As I see it for now, we can specify url which to point v8 repo, and move depot_tools to resources section, which will be a valid approach. Then fetch v8 with depot tools (or take it from cache), sync and update cache. After that we can resume standard build process.